lib/manager/sbt/index.js

```diff
@@ -0,0 +1,8 @@
+const { extractPackageFile } = require('./extract');
+const { updateDependency } = require('./update');
+
+module.exports = {
+  language: 'scala',
```

Comment:
I think we need to add a `scala` object to config definitions if this is present.

Reply:
I've removed this field, it seems to be optional in this case